content/blog/2020-07-07-cml-release.md

```diff
@@ -0,0 +1,182 @@
+   tests does not apply in ML. As an example, `+0.72% accuracy` and
+   `-0.35% precision` does not answer the question if the ML model is good or
+   not. Detailed reports with metrics and plots are needed to make a good/bad
+   model discussion
+3. **CPU/GPU resources**. ML training often requires more resources to train
+   then is typical to have in CI/CD runners. CI/CD must be connected with cloud
+   computing instances or Kubernetes clusters for ML training.
+
+## 2. CI/CD for ML is the next step for DVC team
+
+Since the beginning, our motivation has been helping ML teams benefit from
+DevOps. We started DVC because we knew that data management would be a crucial
+bottleneck, and sure enough, DVC was a big step towards making pipelines and
+experiments manageable and reproducible. But conversations with our community
+have brought us to one conclusion again and again: CI/CD for ML is the holy
+grail.
+
+Over the last 3 years, we've reached some big milestones:
+
+1. We built DVC to address the ML data management problem. Recently, we
+   [released DVC 1.0](https://dvc.org/blog/dvc-1-0-release), marking a new and
+   more stable era for our API.
+2. DVC has become a core part of many ML team's daily operations. The latest
+   [ThoughtWorks Technology Radar](https://www.thoughtworks.com/radar/tools)
+   says:
+
+   _"... it [DVC] has become a favorite tool for managing experiments in machine
+   learning (ML) projects. Since it's based on Git, DVC is a familiar
+   environment for software developers to bring their engineering practices to
+   ML practice."_
+
+3. An extraordinary team and community have emerged around DVC:
+   - 15 employees in our organization https://iterative.ai
+   - 100+ open-source contributors to DVC https://github.com/iterative/dvc and
+     another 100+ open-source contributors to docs
+     https://github.com/iterative/dvc.org
+   - 2000+ community members in our Discord https://dvc.org/chat and GitHub
+     issue tracker https://github.com/iterative/dvc
+   - 4000+ regular users of DVC
+
+Now that DVC is maturing, we're ready to take the next step: we want to
+revolutionize ML development processes. We want ML experiments to have greater
+visibility to teammates, shorter feedback loops, and more reproducibility. We
+want teams to spend less time managing their computing resources and
+experiments, and more time building value. The goal is to extend the amazing
+results of DevOps from software development to ML and MLOps.
+
+## 3. Continuous Machine Learning release
+
+Today, we're releasing an open-source project https://CML.DEV to close the gap
+between machine learning and software development practices.
+
+CML is a library of functions used inside CI/CD runners to make ML compatible
+with **GitHub Actions** and **GitLab CI**. We've created functions to:
+
+1. Generate informative reports on every Pull/Merge Request with metrics, plots,
+   and hyperparameters changes.
```

Comment:
> with metrics, plots, and hyperparameters changes

Q: Are we talking about DVC params, metrics, and plots specifically? Or other formats are supported?

If it's DVC, maybe link to those cmd refs. or even say "DVC metrics, ..."

Reply:
> If it's DVC, maybe link to those cmd refs. or even say "DVC metrics, ..."

Or link to `#4-the-relationship-between-cml-and-dvc`

Reply:
no necessary DVC